crates/sui-indexer/src/metrics.rs

```diff
@@ -34,6 +39,8 @@ pub struct IndexerMetrics {
     pub checkpoint_index_latency: Histogram,
     pub checkpoint_objects_index_latency: Histogram,
     pub checkpoint_db_commit_latency: Histogram,
+    // average latency of 1k transactions
```

Comment:
```suggestion
    // average latency of committing a batch of transactions with CHUNK_SIZE 
```

Reply:
actually 1k is not the batch size; by this metric, I want to have an average tx commit latency metric such that when we tune all kinds of parameters including batch size, we can count on this metric to measure if the change is useful for tx ingestion speed. 
I picked 1k b/c it's very likely to fall into 0.1 to 1 second, which is well covered by DB_COMMIT_LATENCY_SEC_BUCKETS.